main.cpp

```diff
@@ -15,22 +15,42 @@ video: Chapter 2 - Part 3
  
  
  1) Write down the names of the 6 major primitive types available in C++  here:
- 
- 
- 
- 
- 
- 
- 
- 
- 
+    
+ 1) int 
+ 2) char
+ 3) float
+ 4) double
+ 5) bool
+ 6) void
  
 2) for each primitive type, write out 3 variable declarations inside the variableDeclaration() function on line 59.
     a) give each variable declaration an initial value
         - just ignore wchar_t. you do not need to declare 3 variables of type 'wchar_t'
         - 'void' is a return type. you do not need to declare 3 variables of type 'void'.
     b) at the end of the function, call ignoreUnused once and pass all of your variables to it. see line 71 for an example
- 
+
+int age = 30;
+int heighinInches = 72;
+int weightInPounds = 180;
+
+char firstInitial = 'J';
+char middleInitial = 'M';
+char lastInitial = 'S';
+
+float pi = 3.14f;
+float e = 2.71f;
+float tau = 6.28f;
+
+double pi = 3.14;
+double e = 2.71;
+double tau = 6.28;
+
+bool isGameOver = false;
+bool isPathClear = true;
+bool isGameWon = true;
+
+ignoreUnused (age, heighinInches, weightInPounds, firstInitial, middleInitial, lastInitial, pi, isGameOver, isPathClear, isGameWon);
```

Comment:
These go inside the function on line 59, and not as comments either.
- [ ] revise